plugins/lombok/lombok-compiler-plugin/src/org/jetbrains/kotlin/lombok/processor/SetterProcessor.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2010-2021 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.lombok.processor
+
+import org.jetbrains.kotlin.descriptors.*
+import org.jetbrains.kotlin.lombok.config.*
+import org.jetbrains.kotlin.lombok.utils.*
+import org.jetbrains.kotlin.name.Name
+import org.jetbrains.kotlin.resolve.descriptorUtil.builtIns
+
+class SetterProcessor(private val config: LombokConfig) : Processor {
```

Comment:
they have quite a few little difference I don't know how to extract and reuse. If I put it to the single GetterSetterProcessor, it would be too many ifs
basically I trade one iteration over properties for more specific code